Error while checking build/O2Physics/o2 for 54344c3ec00c9069af3dabce67c434374cd0d8c1 at 2026-04-11 06:55:
```
## sw/BUILD/O2Physics-latest/log
c++: fatal error: Killed signal terminated program cc1plus
ninja: build stopped: subcommand failed.
```
Full log [here](https://ali-ci.cern.ch/alice-build-logs/AliceO2Group/O2Physics/15700/54344c3ec00c9069af3dabce67c434374cd0d8c1/build_O2Physics_o2/pretty.html).

--- a/PWGDQ/Tasks/muonGlobalAlignment.cxx
+++ b/PWGDQ/Tasks/muonGlobalAlignment.cxx
@@ -58,10 +58,10 @@
 #include <THnSparse.h>
 #include <TMath.h>
 
+#include "rapidjson/document.h"
+
 #include <GPUROOTCartesianFwd.h>
 #include <RtypesCore.h>
-
-#include "rapidjson/document.h"
 
 #include <algorithm>
 #include <array>

